src/SOS/Strike/CMakeLists.txt

```diff
@@ -97,7 +96,6 @@ if(WIN32)
   set(SOS_LIBRARY
     extensions
     corguids
-    debugshim
```

Comment:
Now that this is gone from here and debugshim was a static lib, is there anything using it at all? Can it be deleted?